### PR TITLE
Add necessary modification for the retention LP

### PR DESCRIPTION
**Pull request by Matei-Iordache**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--helix-poc--enake.hlx.page/consumer/en/renew/poy2022-offline-active-ts-av-is
- After: https://retention-LP--helix-poc--enake.hlx.page/consumer/en/renew/poy2022-offline-active-ts-av-is

### CODE REVIEW

**Review by Tiseanu — APPROVED** [review]

File: blocks/b-banner/b-banner.css [hunk 1]
```diff
@@ -48,6 +48,17 @@
   max-width: 595px;
 }
```
Comment (vladalexandruradulescu) [inline blocks/b-banner/b-banner.css:50]: font-size and position of the image is not the same as per "design" ( https://www.bitdefender.com/media/html/consumer/renew/poy2022-offline-active-ts-av-is/ )

File: blocks/b-banner/b-banner.js [hunk 2]
```diff
@@ -50,6 +59,29 @@ export default function decorate(block) {
...
+    image.style.height = '-webkit-fill-available';
+    image.style.objectFit = 'cover';
+  }
+
+  if (hasAwardImage !== 'undefinded') {
+    const bBanner = document.querySelector('.b-banner');
+    bBanner.classList.add('d-flex');
+
+    const AwardImage = block.querySelectorAll('picture')[0];
+    const AwardImage2 = AwardImage.querySelector('img');
```
Comment (vladalexandruradulescu) [inline blocks/b-banner/b-banner.js:77]: other LPs have errors, for example: https://retention-lp--helix-poc--enake.hlx.page/business/nl/ransomwaretrial

File: blocks/c-productswithvpn/c-productswithvpn.scss [hunk 3]
```diff
@@ -170,6 +174,11 @@
   color: #000;
   text-align: left;
   margin-top: 10px;
+
+  &.check-full {
+    background: url("../../images/icons/check_full.png") 0 3px no-repeat transparent;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn/c-productswithvpn.scss:179]: we have double support functionality in place to make use of svg files, no need for this approach anymore
examples are: 
1) ( dynamic content ) header.js witch reads the nav.docx => finds usage of logo-white.svg => app knows how to render logo-white.svg/
2) static / fixed content: use **SvgLoaderComponent**

File: blocks/b-banner/b-banner.css (hunk 1 above)
Comment (vladalexandruradulescu) [inline blocks/b-banner/b-banner.css:50]: block rendering was modified: https://docs.google.com/document/d/1Vg4QdD0lVGooZoTkEja01o3cvq1brnSFaW6tTAeLdb8/edit

please be aware of braking changes in others LPs

File: blocks/b-banner/b-banner.js [hunk 4]
```diff
@@ -26,18 +26,27 @@ export default function decorate(block) {
...
+  bannerImage.classList.add('b-banner-hero-image');
   parentSelector.append(bannerImage);
 
   // config new elements
   const paragraph = block.querySelector('p');
+  const banner = document.querySelector('.b-banner-container');
   const {
-    product, discountStyle, discountText, backgroundColor, imageVariation,
+    product, discountStyle, discountText, backgroundColor, imageVariation, paragraphColor,
+    bannerHeight, bannerImageCover, hasAwardImage, awardImageWidth, bannerColor,
```
Comment (vladalexandruradulescu) [inline blocks/b-banner/b-banner.js:39]: has_award_image is to specific ... maybe has_preamble and preamble_position

**Pull request closed without merging**